src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Performance Tooling Score Differences
+subhead: I get different scores in Test My Site, PageSpee Insights, and Lighthouse - why?
+authors:
+  - egsweeny
+date: 2019-07-05
+hero: hero.jpg
```

Comment:
Post needs a hero image. You can find good public domain images on unsplash.com. You'll want to crop the image to 1600 x 800. I'm happy to do this if you find an image you like but don't have photoshop setup yet.

Reply:
I added a hero image, but imagine that it isn't in the right size. Thank you in advance for tweaking if it isn't... :)